packages/scandipwa/src/component/ProductWishlistButton/ProductWishlistButton.container.js

```diff
@@ -98,7 +99,7 @@ export class ProductWishlistButtonContainer extends PureComponent {
         return this.setState({ isWishlistButtonLoading: isLoading });
     }
 
-    async toggleProductInWishlist(add = true) {
+    async toggleProductInWishlist(add) {
```

Comment:
There is a reason for it?

Reply:
In my opinion, this is not always safe, if somehow the value does not come to the function, then another error may pop up. Maybe you need to add a check for the value, whether it exists or not. Or maybe I'm wrong)